app/move/controllers/create/move-details.test.js

```diff
@@ -197,86 +197,110 @@ describe('Move controllers', function() {
             locationsStub.resetHistory()
           })
 
-          it('when the current location type is prison appends to_location_prison_recall as conditional', async function() {
-            req.session = {
-              currentLocation: {
-                location_type: 'prison',
-              },
+          context(
+            'when the current location type is prison appends to_location_prison as conditional',
```

Comment:
You could remove the text that now sits in the `if` texts. The context is effectively just saying what the condition is for that group of tests, so in this case: `when the current location type is prison`.

You can also drop the  `async` as there is no `await` used within this function anymore.